aleph-client/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aleph_client"
-version = "3.5.0"
+version = "3.5.1"
```

Comment:
You are adding at least one new functionality (timestamps), so bump the minor version.

Reply:
Done